### PR TITLE
File name inconsistent

### PR DESCRIPTION
**Pull request by ddjnw1yu**
# Description

The PR is going to fix the segmentation part of the issues mentioned in tickets - [Bug Submission: segmentation viewer returning an error e.g. dataset 125](https://www.wrike.com/open.htm?id=1358702730) and [Bug Submission: segmentation viewer and plot viewer preview not available e.g. dataset 119 and 125](https://www.wrike.com/open.htm?id=1358702570). Inconsistent file names are stored between scicrunch and neurolucida, scicrunch and s3.

## Type of change

Delete those that don't apply.

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Run the portal locally to check whether the thumbnail is showing.
Screenshot 2024-06-27 at 9 37 19 AM

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added unit tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Review by hsorby — DISMISSED** [review]

File: app/manifest_name_to_discover_name.py [hunk 1]
```diff
@@ -5133,6 +5133,67 @@
...
+            'files/derivative/sub-M16/sam-pCm16/digital-traces/pC PHPS XFP M16 20XZ 180425_180524.xml': 'files/derivative/sub-M16/sam-pCm16/digital-traces/pC_PHPS_XFP_M16_20XZ_180425_180524.xml',
+            'files/derivative/sub-M163/sam-pCm163/digital-traces/pCm163_AAV_Z_20x_20200206_S3_F peach.png.xml': 'files/derivative/sub-M163/sam-pCm163/digital-traces/pCm163_AAV_Z_20x_20200206_S3_N12_connex.xml',
+            'files/derivative/sub-M163/sam-pCm163/digital-traces/pCm163_AAV_Z_20x_20200206_S3_N12 connex.xml': 'files/derivative/sub-M163/sam-pCm163/digital-traces/pCm163_AAV_Z_20x_20200206_S3_N12_connex.xml',
+            'files/derivative/sub-M168/digital-traces/pCm168_AAV_Z_20x_191211_S3B_lx_F beige.xml': 'files/derivative/sub-M168/digital-traces/pCm168_AAV_Z_20x_191211_S3B_lx.xml',
+            'files/derivative/sub-M168/digital-traces/pCm168_AAV_Z_20x_191211_S3B_lx_F G.xml': 'files/derivative/sub-M168/digital-traces/pCm168_AAV_Z_20x_191211_S3B_lx.xml',
+            'files/derivative/sub-M168/digital-traces/pCm168_AAV_Z_20x_191211_S3B_lx_IG connex.xml': 'files/derivative/sub-M168/digital-traces/pCm168_AAV_Z_20x_191211_S3B_lx.xml',
```
Comment (hsorby) [inline app/manifest_name_to_discover_name.py:5189]: These files are all pointing to the same place, they also seem wrong to me given what is in the dataset.

**Review by hsorby — COMMENTED** [review]
Can these entries be double checked, on the surface they don't seem correct (and  maybe others).

File: app/manifest_name_to_discover_name.py (hunk 1 above)
Comment (ddjnw1yu) [reply app/manifest_name_to_discover_name.py:5189]: I see. A minor issue with my script. I will double-check this.

**Comment by ddjnw1yu** [comment]
An if statement is accidentally removed from my script. I have updated and double-checked the file names. They should all match now.

**Review by hsorby — APPROVED** [review]

**Pull request merged**